### DISCUSSION TITLE
Duda datasets proyecto final

### DISCUSSION THREAD

**Discussion by diegosaaval** [rramosp/ai4eng.v1#152] [UDEA Students]
Buen día profesor y monitor, espero se encuentren muy bien

Quisiera saber cuál de estos datasets son apropiados para el proyecto final, quedo atento a la respuesta y muchas gracias

https://www.kaggle.com/competitions/competitive-data-science-predict-future-sales/data

https://www.kaggle.com/competitions/spaceship-titanic/data

https://www.kaggle.com/datasets/mitishaagarwal/patient

**Comment by rramosp** [comment]
si cumple las condiciones que pusimos, pues sí ... si no las cumple, piensa
cómo lo suplirías como se explica en la página del grupo (simulando datos
faltantes, columnas nuevas, etc.)

On Thu, Jun 30, 2022 at 7:46 AM DiegoSaav ***@***.***> wrote:

> Buen día profesor y monitor,
>
> Quisiera saber si este dataset es apropiado para el proyecto final, quedo
> atento a la respuesta y muchas gracias
>
> https://www.kaggle.com/competitions/spaceship-titanic/data
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/152>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIL66NKOSZD772JPLTDVRWJJFANCNFSM52JHM25A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
